facia/app/controllers/FaciaController.scala

```diff
@@ -352,27 +353,62 @@ trait FaciaController
 
   def renderShowMore(path: String, collectionId: String): Action[AnyContent] =
     Action.async { implicit request =>
-      frontJsonFapi.get(path, fullRequestType).flatMap {
-        case Some(pressedPage) =>
-          val containers = Front.fromPressedPage(pressedPage, Edition(request), adFree = request.isAdFree).containers
-          val maybeResponse =
-            for {
-              (container, index) <- containers.zipWithIndex.find(_._1.dataId == collectionId)
-              containerLayout <- container.containerLayout
-            } yield {
-              val remainingCards: Seq[FaciaCardAndIndex] = containerLayout.remainingCards.map(_.withFromShowMore)
-              val adFreeFilteredCards: Seq[FaciaCardAndIndex] = if (request.isAdFree) {
-                remainingCards.filter(c => !checkIfPaid(c.item))
-              } else {
-                remainingCards
+      if (request.forceDCR) {
+        frontJsonFapi.get(path, fullDCRRequestType).flatMap {
+          case Some(pressedPage) =>
+            val matchedCollection = pressedPage.collections.find(_.id == collectionId)
+
+            // TODO: length is Option because not all types of pressedCollection have the property, but
+            //  pressedCollections coming from *DCR* pressedPages *should* have the length prop. If they don't
+            //  then something has gone wrong. How should we surface this? (Default value of 0 avoids a crash
+            //  but it covers over the bug.)
+            matchedCollection
+              .map(collection => {
+
+                val liteCuratedLength = collection.liteCuratedLength.getOrElse(0)
+                val liteBackfillLength = collection.liteBackfillLength.getOrElse(0)
+
+                val pressedContent = collection.curated.takeRight(
+                  collection.curated.length - liteCuratedLength,
+                ) ++ collection.backfill.takeRight(collection.backfill.length - liteBackfillLength)
+                val startIndex = liteCuratedLength + liteBackfillLength
+
+                val config = collection.config
+
+                remoteRenderer.getCards(ws = ws, cards = pressedContent, startIndex = startIndex, config = config)
+
+              }) getOrElse successful(Cached(CacheTime.NotFound)(WithoutRevalidationResult(NotFound)))
+
+          // TODO we also need to handle the case where this doesn't work
+          //  - Are there importantly different failure cases to handle, or can we return a generic fallback response,
+          //    given that DCR doesn't really do anything with the response in the case of failure?)
+
+          case None => successful(Cached(CacheTime.NotFound)(WithoutRevalidationResult(NotFound)))
+        }
+      } else {
+        frontJsonFapi.get(path, fullRequestType).flatMap {
```

Comment:
the contents of this `else` block are (or should be) the same as the existing Frontend endpoint, but git isn't showing this very clearly